src/lib/Components/Artist/Header.tsx

```diff
@@ -141,6 +125,35 @@ class Header extends React.Component<HeaderProps, State> {
 
     return leadingSubstring + " " + birthday
   }
+
+  @track({ action: "press follow/unfollow button" })
```

Comment:
This could use the props/state to reflect the correct follow/unfollow caption, right?

Reply:
good catch! code predates state addition :D. fixed now